.github/actions/create-release-archive/action.yml

```diff
@@ -35,4 +35,4 @@ runs:
 
     - name: Make archive
       shell: bash
-      run: tar -C ./target/${{ inputs.target }}/release --create --file=${{ steps.create-archive-name.outputs.archive }} ${{ inputs.binary }}
\ No newline at end of file
+      run: gtar -C ./target/${{ inputs.target }}/release --create --file=${{ steps.create-archive-name.outputs.archive }} ${{ inputs.binary }}
```

Comment:
I assume you tested that this actually works on all platforms? i.e. `windows-latest` actually ships with a `gtar` binary?

Reply:
it doesn't : (. Had to create a new action